lib/releaselib.sh

```diff
@@ -446,12 +441,10 @@ release::set_release_version () {
     RELEASE_VERSION_PRIME="${RELEASE_VERSION[alpha]}"
   fi
 
-  if ((FLAGS_verbose)); then
-    for label in ${!RELEASE_VERSION[*]}; do
+  for label in ${!RELEASE_VERSION[*]}; do
```

Comment:
```suggestion
  for label in ${!RELEASE_VERSION[@]}; do
```

Same as other review comment.

Reply:
There's one other ${!RELEASE_VERSION[*]} in the codebase which might merit changing later too.  Shellcheck all the things has proved ... complex though.